types/express-serve-static-core/index.d.ts

```diff
@@ -71,8 +71,10 @@ export interface IRouterMatcher<T, Method extends 'all' | 'get' | 'post' | 'put'
 }
 
 export interface IRouterHandler<T> {
-    (...handlers: RequestHandler[]): T;
-    (...handlers: RequestHandlerParams[]): T;
+  // tslint:disable-next-line no-unnecessary-generics (This generic is meant to be passed explicitly.)
+  <P = ParamsDictionary, ResBody = any, ReqBody = any, ReqQuery = ParsedQs>(...handlers: Array<RequestHandler<P, ResBody, ReqBody, ReqQuery>>): T;
+  // tslint:disable-next-line no-unnecessary-generics (This generic is meant to be passed explicitly.)
```

Comment:
Are these lines required or just a copy-paste from above? I'm not clear on why they're considered "unnecessary" since they are used in the reference to `RequestHandler`.

Reply:
I did omit them initially and the tests failed without them. 

I've just checked again to be safe and without them the tests have following output:
```
ERROR: 74:111  no-unnecessary-generics  Type parameter P is used only once. See: https://github.com/Microsoft/dtslint/blob/master/docs/no-unnecessary-generics.md
ERROR: 74:114  no-unnecessary-generics  Type parameter ResBody is used only once. See: https://github.com/Microsoft/dtslint/blob/master/docs/no-unnecessary-generics.md
ERROR: 74:123  no-unnecessary-generics  Type parameter ReqBody is used only once. See: https://github.com/Microsoft/dtslint/blob/master/docs/no-unnecessary-generics.md
ERROR: 74:132  no-unnecessary-generics  Type parameter ReqQuery is used only once. See: https://github.com/Microsoft/dtslint/blob/master/docs/no-unnecessary-generics.md
ERROR: 75:117  no-unnecessary-generics  Type parameter P is used only once. See: https://github.com/Microsoft/dtslint/blob/master/docs/no-unnecessary-generics.md
ERROR: 75:120  no-unnecessary-generics  Type parameter ResBody is used only once. See: https://github.com/Microsoft/dtslint/blob/master/docs/no-unnecessary-generics.md
ERROR: 75:129  no-unnecessary-generics  Type parameter ReqBody is used only once. See: https://github.com/Microsoft/dtslint/blob/master/docs/no-unnecessary-generics.md
ERROR: 75:138  no-unnecessary-generics  Type parameter ReqQuery is used only once. See: https://github.com/Microsoft/dtslint/blob/master/docs/no-unnecessary-generics.md
```